draft-ietf-mops-streaming-opcons.md

```diff
@@ -380,23 +391,16 @@ bandwidth estimation/prediction and the bitrate selection. Most players only use
 This kind of bandwidth-measurement systems can experience trouble in
 several ways that can be affected by networking design choices.
 
-###Idle Time between Segments
+### Idle Time between Segments
 
-When the bitrate selection is successfully chosen below the
-available capacity of the network path, the response to a
-segment request will typically complete in less absolute time than the
-duration of the requested segment. The resulting idle time within the connection carrying the
-segments has a few surprising consequences:
+When the bitrate selection is chosen substantially below the available capacity of the network path, the response to a segment request will typically complete in much less absolute time than the duration of the requested segment, leaving significant idle time between segment downloads.  This can have a few surprising consequences:
 
- * Mobile flow-bandwidth spectrum and timing mapping.
+ * TCP slow-start when restarting after idle requires multiple RTTs to re-establish a throughput at the network's available capacity.  When the active transmission time for segments is substantially shorter than the time between segments leaving an idle gap between segments that triggers a restart of TCP slow-start, the estimate of the successful download speed coming from the application-visible receive rate on the socket can thus end up much lower than the actual available network capacity, preventing a shift to the most appropriate bitrate.  {{RFC7661}} provides some mitigations for this effect at the TCP transport layer, for senders who anticipate a high incidence of this problem.
 
- * TCP slow-start when restarting after idle requires multiple
-   RTTs to re-establish a throughput at the network's available
-   capacity. On high-RTT paths or with small enough segments,
-   this can produce a falsely low application-visible measurement
-   of the available network capacity.
+ * Mobile flow-bandwidth spectrum and timing mapping can be impacted by idle time in some networks.  The carrier capacity assigned to a link can vary with activity. Depending on the idle time characteristics, this can result in a lower available bitrate than would be achievable with a steadier transmission in the same network.
 
-A detailed investigation of this phenomenon is available in {{NOSSDAV12}}.
+A detailed investigation of these phenomena is available in {{NOSSDAV12}}.
```

Comment:
s/A detailed investigation of these phenomena is available in {{NOSSDAV12}}./How TCP slow-start restart can be avoided by the help of two simultaneous TCP connections is exemplified for Microsoft Smooth Streaming in {{MMSys11}}.

MMSys11: Saamer Akhshabi, Ali C. Begen and Constantine Dovrolis, "An experimental evaluation of rate-adaptation algorithms in adaptive streaming over HTTP," in Proc. ACM Multimedia Systems Conf. (MMSys), San Jose, CA, Feb. 2011 (DOI=10.1145/1943552.1943574)